### DISCUSSION TITLE
Scope Preflight?

### DISCUSSION THREAD

**Discussion by chardin1** [tailwindlabs/tailwindcss#9724] [Help]
Is there an option to scope Preflight to a particular class or set of classes? I don't want to remove the base styles but only want them to apply to a certain area.

**Comment by wongjn** [comment]
You could [disable the core preflight plugin](https://tailwindcss.com/docs/preflight#disabling-preflight) and then emulate what [the core prelight plugin does](https://github.com/tailwindlabs/tailwindcss/blob/226be119dfb95706486df0d38f3ce357fb4ac856/src/corePlugins.js#L462-L473) in your own plugin but with your own scope.
